ccvm_simulators/problem_classes/boxqp/tests/test_problem_instance.py

```diff
@@ -97,19 +97,48 @@ def test_load_instance_cpu(self):
             file_path=self.file_path,
             file_delimiter="\t",
         )
-
-        expected_optimal_sol = 799.560976
+        
+        '''
+        self.device = device
+        self.instance_type = instance_type
+        self.problem_size = problem_size
+        self.optimal_sol = optimal_sol
+        self.best_sol = best_sol
+        self.optimality = optimality
+        self.sol_time_gb = sol_time_gb
+        self.sol_time_bfgs = sol_time_bfgs
+        self.num_frac_values = num_frac_values
+        self.q_matrix = rval_q
+        self.v_vector = rval_v
+        self.optional_info = optional_info
```

Comment:
I think this should be solution_vector now?

Reply:
Thanks Katie, I removed the lines between 101-113 which was actually a commented segment that I copied from problem_instance.py but forgot to delete. Now it is done.